lib/hanami/components/components.rb

```diff
@@ -29,7 +29,7 @@ module Components # rubocop:disable Metrics/ModuleLength
 
       resolve do |configuration|
         if configuration.logger.is_a?(Array)
-          if configuration.logger.first.is_a?(::Logger)
+          if configuration.logger.first.is_a?(::Logger) || logger_interface?(configuration.logger.first)
```

Comment:
Do we even need the `is_a?(::Logger)` check if the `logger_interface?` method will return true for `::Logger` objects anyway?

Reply:
Yep, we can use this method for `::Logger` too. That's why we can drop `is_a?(::Logger)` 🎉